hack/run-all.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Run all the important scripts in a sensible order,
+# handy for switching branches, etc
+wd="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
Heh, well, I used it without being able to explain it (bad).  Had to look it up.